scripts/activation_responses.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
```

Comment:
Is there a way to merge this with `r2s.py` or pull out some common elements?  It seems that the two files are very similar

Reply:
Maybe that could be left for another PR??

Reply:
Yes, this file and the `r2s.py` shares almost the same work flow. We can combine them together. However, I don't think it's a good idea to merge them at the current stage.
Because some variables work differently in the two files. For example:
- the `sub-voxel` option could not used in `activation_responses.py` now;
- the `photon_source` meaning different thing in `r2s.py` (multiple group` and `activation_responses.py` (total`)

I think it's better to merge these two files when every thing is done with this PR, and we need also wait for the changes related to DagOpenMC R2S.